include/swift/Frontend/Types.h

```diff
@@ -19,11 +19,10 @@
 #include <functional>
 
 namespace swift {
-namespace driver {
 namespace types {
```

Comment:
Hm. `swift::types` makes me think of types in the language rather than file types. Maybe we should rename the namespace to `file_types`?

(yes, it's weird that [namespaces are the only thing in the project that uses lower_snake_case](https://github.com/apple/swift/blob/master/include/swift/Basic/StringExtras.h#L70))

Reply:
Agree! Separate PR? Before or after this one??

Reply:
Ah, I figured that since you were removing `driver` most of the use sites would have to be updated anyway, but that's probably not true. "Before" is probably the most correct thing to do, but either way is fine as long as we don't forget about it.

Reply:
OK, tx. Will do it after--really want to land this one (as usual).